docs/how-to-restart-server.md

```diff
@@ -1 +1,12 @@
-# How do I restart my stardew server?
+# How do I restart my Junimohost Stardew server?
+
+We understand that occasional bugs can hinder your gameplay, but fret not – we've got a quick and easy solution for you. The best place to start when troubleshooting issues with your Junimohost server is to restart the server. This can be accomplished by following the below steps.
+
+What are the steps? (PC)
+1. Visit junimohost.com to access your account.
```

Comment:
linking to junimohost.com/dashboard will automatically redirect to the login if they arent already logged in, that reduces the first 2 steps down to 1.